.buildkite/configs/cleanup.aws.yml

```diff
@@ -0,0 +1,75 @@
+---
+version: "1.0"
+
+accounts:
+  - name: "${ACCOUNT_PROJECT}"
+    driver: "aws"
+    options:
+      key: '${ACCOUNT_KEY}'
+      secret: '${ACCOUNT_SECRET}'
+
+scanners:
+  - account_name: "${ACCOUNT_PROJECT}"
+    resources:
+      - type: 'node'
+        regions:
+          - us-east-1
```

Comment:
Do we have the region in some environment variable that we could use here?

Reply:
Currently, it is using the default region set in the terraform files, there is no value set in our scripts. For instance:
https://github.com/elastic/elastic-package/blob/e74c0219d45746fba0a6340cfefcda88509791f0/test/packages/parallel/aws/data_stream/ec2_metrics/_dev/deploy/tf/env.yml#L9

https://github.com/elastic/integrations/blob/4efd95037467d5e54270006bcd71c3b0b6982932/packages/cisco_umbrella/_dev/deploy/tf/env.yml#L9

Should we add a value there in our scripts for both repositories or rely on that default value?
Currently, all terraform tags are set in the `pre-command` hook:
https://github.com/elastic/elastic-package/blob/346081a12249f64ff4b523effb68888835d8644c/.buildkite/hooks/pre-command#L14

Maybe, we could add/set there the variable `export AWS_REGION=us-east-1` too.